apps/discovery_streams/config/config.exs

```diff
@@ -15,7 +15,16 @@ config :logger,
 
 config :discovery_streams,
   ttl: 600_000,
-  topic_prefix: "transformed-"
+  topic_prefix: "transformed-",
+  topic_subscriber_config: [
+    begin_offset: :latest,
+    offset_reset_policy: :reset_to_latest,
+    max_bytes: 10_000_000,
+    min_bytes: 0,
+    max_wait_time: 10_000,
+    prefetch_count: 0,
+    prefetch_bytes: 100_000_000
```

Comment:
have we tested how these kinds of things perform for datasets which trickle a few bytes in a minute?

Reply:
For intermittent, small chunks of data, we have not. There is not much that can be done here to help prefetch with that. WRT to max bytes and wait time, the min_bytes of 0 has some special kafka magic that will still forward on trickles of data. It's like an internal threshold of 100ms or something. I'll go dig it up. But yeah, as long as `min_bytes` is set to 0, we don't end up having to hit the max wait time for trickles